### ISSUE TITLE
AE2 and radioactive

### ISSUE THREAD

**Issue by Tsunayoshi** [jakimfett/Minechem#710]
Hey,

I have some problems with AE2 and the radioactive elements. The decay ticks in the autocrafting pattern, therefor AE2 can't autocraft radioactive materials.

My Setup:
- I have decomposed blutonium (purified cyanite) from big reactors mod.
- Here was my fist problem. If I had import the Pu94 into my AE2 I can't extract it from the AE system because the decay ticks
- So I piped it into the fisson chamber to get Ag47, as log as I don't look into the fisson chamber the timer doesn't start.
- Now I have teached my AE System to craft all along to U92 with the last step 47+45
- Here is my second problem, that I can't solve without an extra fusion chamber - only for the last step... An export bus with crafting and fuzzy upgrade doesn't work. The system doesn't start the crafting. If I start the crafting by hand, it works... but! the AE doesn't realise the incoming 92 because the decay ticks in the crafting pattern, so the new U92 isn't the same as in the pattern.
- Now I am stuck :p

Is it intend to stop make the autocrafting for radioactive elements this hard? Or is it planed in futur to support the AE autocrafting?

btw: I am storing all radioactive elements in Lead Chest with a storage bus. If I look into my AE Terminal all timers are active. If I take out a element that has the (unstable)-Tag, it kills me, but if I look into the lead chest and then into the Terminal the time is reset and start form new.

MC 1.7.10
forge 1291
Minechem-1.7.10-5.0.5.393
appliedenergistics2-rv2-beta-14

**Comment by way2muchnoise** [comment]
@Tsunayoshi The issue for all those things are with AE. They don't follow the regular minecraft ways so would have to support every single line in there code. We made some fixes for pattern things (see e08a9362074255bf29b5fdaa4702f17bb2b5a4a5) but those only work for stacks with size zero which is common in filters and such. So there is no real way to just fix it.

For the decaying it is the same deal AE does some magics where we don't have control so the default tick handler takes over. When using storage busses for an AE point it gets inserted to AE so it no longer is in the chest while we check if it is in a leaded chest which for that code scope isn't so that is why it decays.

TLDR: There is no easy way to support all of AE we did our best for some auto crafting ways but we can't cover it all. It isn't intended to make autocrafting hard/impossible but is the reality.

**Comment by Tsunayoshi** [comment]
Ah ok, thanks for your quick answer!
I think this here can be closed because, there is a way to do it. My AE Sys can already craft the 47 and 45, so I think I will build another fusion chamber only for the U92. This only needs more room, so I don't see any problem with this. I was only interessed if i am doing anything worng here ;)

Nice work! Your mod has become one of my fav. in the last two months. keep going.

**Comment by way2muchnoise** [comment]
@Tsunayoshi That is why we are here. Glad you love the mod. And get hyped for v6! Also you can always finds at IRC on esper.net in #minechem if you have any questions or just want to hang out with us.
